configure.ac

```
@@ -1135,11 +1136,13 @@ AS_IF([test "x$with_openssl" = "xyes"],[
     SQUID_CHECK_OPENSSL_CONST_SSL_SESSION_CB_ARG
     SQUID_CHECK_OPENSSL_CONST_X509_GET0_SIGNATURE_ARGS
     SQUID_CHECK_OPENSSL_TXTDB
+    AC_PATH_PROG(OPENSSL_BIN, openssl)
   ])
   AS_IF([test "x$SSLLIB" = "x"],[AC_MSG_ERROR([Required OpenSSL library not found])])
 ])
 AC_MSG_NOTICE([OpenSSL library support: ${with_openssl:=no} ${LIBOPENSSL_PATH} ${LIBOPENSSL_LIBS}])
 AM_CONDITIONAL(ENABLE_SSL,[ test "x$with_openssl" = "xyes" ])
+AM_CONDITIONAL(ENABLE_CERT_TESTS,[ test "x$with_openssl" = "xyes" -o "x$with_gnutls" != "xno"])
 AC_SUBST(SSLLIB)
```

Please move this line down and visually separate it from OpenSSL-specific code. It is not specific to OpenSSL. It covers both GnuTLS and OpenSSL cases (handled above).

```suggestion
AC_SUBST(SSLLIB)

AM_CONDITIONAL(ENABLE_CERT_TESTS,[ test "x$with_openssl" = "xyes" -o "x$with_gnutls" != "xno"])
```